sdk/examples/attestation/findBy.ts

```diff
@@ -0,0 +1,5 @@
+import VeraxSdk from "../../src/VeraxSdk";
+
+const veraxSdk = new VeraxSdk(VeraxSdk.DEFAULT_LINEA_TESTNET);
+
+console.log(await veraxSdk.attestation.findBy());
```

Comment:
* arguments missing
* implementation missing
* service is `attestationDataMapper` and not `attestation`

Reply:
* Arguments for this call will be covered in #172 
* `findBy` is implemented and working, even it works like a `findAll` at the moment
* The service was renamed to `attestation` for now, I felt like `attestationDataMapper` is quite long/opinionated for a dev, wdyt?